VTEX - Payment Provider Protocol.json

```diff
@@ -1718,112 +2141,58 @@
                 "deprecated": false
             }
         },
-        "/payments/{paymentId}/inbound-request/{action}": {
+        "/payments/{paymentId}/inbound/{action}": {
             "post": {
                 "tags": [
                     "Payment Flow"
                 ],
                 "summary": "Inbound Request (BETA)",
-                "description": "Forwards a request back to your endpoint using the `inboundRequestsUrl` provided in the POST `/payments` payload.\r\n\r\nThe `{action}` part of the path will be the same you have used to call the `inboundRequestsUrl`.",
+                "description": "Forwards a request back to your endpoint using the `inboundRequestsUrl` provided in the [Create Payment](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments?endpoint=post-/payments) endpoint payload.\r\n\r\nThe `{action}` part of the path will be the same you have used to call the `inboundRequestsUrl`. \r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Maybe I lack the necessary context, but I could not understand what is the purpose of this endpoint. I suggest including an example use case.

Reply:
Boa. Adicionei novo conteúdo após alinhar informações com o Henrique e também verificar o artigo [Implementing a Payment Provider](https://developers.vtex.com/docs/guides/payments-integration-implementing-a-payment-provider).